<?xml version="1.0" ?>
<launch>    
    <param name="gazebo"                   value="false"    type="bool"/>
    <param name="gazebo_robot_name"        value="rh_p12_rn"/>  
    <param name="offset_file_path"         value="$(find rh_p12_rn_manager)/config/offset.yaml"/>
    <param name="robot_file_path"          value="$(find rh_p12_rn_manager)/config/RH-P12-RN.robot"/>
    <param name="init_file_path"           value="$(find rh_p12_rn_manager)/config/dxl_init.yaml"/>
    <param name="device_name"              value="/dev/ttyUSB0"/>
    <!-- RH-P12-RN Manager -->
    <node pkg="rh_p12_rn_manager" type="rh_p12_rn_manager" name="rh_p12_rn_manager" output="screen"/>
</launch>
